src/main/java/com/hivemq/adapter/sdk/api/config/PollingContext.java

```diff
@@ -26,12 +26,12 @@ public interface PollingContext {
     /**
      * @return the topic on which the data will be available
      */
-    @Nullable String getDestinationMqttTopic();
+    @NotNull String getMqttTopic();
 
     /**
      * @return the Quality-of-Service for the MQTT publish containing the data
      */
-    int getQos();
+    int getMqttQos();
```

Comment:
Could this be bound to an enum?

Reply:
absolutely, but this PR is only about config and naming streamline and imho big enough. We can go over the SDK at another point and check the actual returnvalues, we have to anyway when we implement the other direction.

Reply:
Ok, sorry, added the same comment in the other places too.
Should I create a ticket and do that change from there?

Reply:
that would be great yeah